VTEX - Checkout Configuration API.json

```diff
@@ -0,0 +1,350 @@
+        "version": "1.0.0"
+    },
+    "servers": [
+        {
+            "url": "https://{accountName}.{environment}.com.br",
+            "description": "VTEX server URL.",
+            "variables": {
+                "accountName": {
+                    "description": "Name of the VTEX account. Used as part of the URL",
+                    "default": "apiexamples"
+                },
+                "environment": {
+                    "description": "Environment to use. Used as part of the URL.",
+                    "enum": [
+                        "vtexcommercestable"
+                    ],
+                    "default": "vtexcommercestable"
+                }
+            }
+        }
+    ],
+    "paths": {
+        "/api/checkout/configuration/ui": {
+            "get": {
+                "tags": [
+                    "Configuration"
+                ],
+                "summary": "Get checkout configuration",
+                "description": " Retrieves a single configuration of the checkout process for the store.",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "name": "an",
+                        "in": "query",
+                        "description": "Checkout configuration ID.",
+                        "required": true,
+                        "schema": {
+                            "type": "string",
+                            "example": "12872182"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Successful operation",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Settings"
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Configuration not found"
+                    }
+                }
+            },
+            "post": {
+                "tags": [
+                    "Configuration"
+                ],
+                "summary": "Create a new checkout configuration",
+                "description": "Creates a new checkout configuration for the store. This endpoint allows stores to define and set up a custom checkout configuration to their specific requirements.",
```

Comment:
What kind of custom configuration? Is there a guide explaining the possibilities?